hycom/RELO/src_2.2.98ZA-07Tsig0-i-sm-sse_relo_mpi/mod_momtum.F

```diff
@@ -43,6 +43,7 @@ subroutine momtum_hs(m,n)
 #ifdef CPL_OASIS_HYCOM
       use mod_cpl_oasis_init
 #endif
+      use mod_NERSCnml, only : relative_wnd_coef, nersc_init
```

Comment:
nersc_init is not used here. No need to include it. It is read in just after blkdat.